python/cudf/cudf/core/series.py

```diff
@@ -651,7 +651,11 @@ def func(lhs, rhs):
                     rhs = rhs.fillna(fill_value)
                     result = func(lhs, rhs)
                     data = column.build_column(
-                        data=result.data, dtype=result.dtype, mask=mask
+                        data=result._column.base_data,
+                        dtype=result.dtype,
+                        mask=mask,
```

Comment:
mask is calculated above but could have originally been from an offsetted column and had the offset nullified during computation.

Either need to go above and use `lhs._column.base_mask` or pass `.data` instead of `.base_data`

Reply:
Else will leave it as .data and remove offset